### PR TITLE
update_automation: drop symlinks

### PR DESCRIPTION
**Pull request by jdsn**
Instead jenkins jobs either need to call mkcloud from the real path
or have the checkout dir in $PATH

### CODE REVIEW

File: scripts/jenkins/update_automation [hunk 1]
```diff
@@ -33,11 +33,4 @@ else
     popd > /dev/null
 fi
 
-for sc in $@ $(basename $0) ; do
-    find $a_dir -type f -name "${sc}" | while read scriptname ; do
-        ln -s -f ~/${scriptname} ~/bin/$sc
```
Comment (bmwiedemann) [inline scripts/jenkins/update_automation]: How about something like echo "exec ../${scriptname} \"\$@\"" > ~/bin/$sc

Comment (jdsn) [inline scripts/jenkins/update_automation]: why an `exec` here? this does not make sense at all

Comment (jdsn) [inline scripts/jenkins/update_automation]: the overall goal is simplification and not to make it more complex

**Comment by jdsn** [comment]
The recreation of the symlinks caused breakages today again. We need to urgently merge this.

**Comment by vuntz** [comment]
+1

**Comment by rsalevsky** [comment]
+1

**Pull request merged**